components/specification/released-schema/2016-DEV0/ome.xsd

```diff
@@ -3882,7 +3885,7 @@ PlaneInfo (as Plane)
       <xsd:complexContent>
         <xsd:extension base="Shape">
           <xsd:sequence>
-            <xsd:element ref="BinData" minOccurs="1" maxOccurs="unbounded"/>
+            <xsd:element ref="BinData" minOccurs="1" maxOccurs="1"/>
```

Comment:
If that makes the most sense then it is an easy change from the model code gen perspective. I had left it for now as it matched up with the api calls unlike for Mask, an update to apply the restriction on Pixels would change the API functions to remove the binDataIndex parameter.